tools/validator-client-scripts/Makefile

```diff
@@ -0,0 +1,8 @@
+install:
```

Comment:
Cant we put those `make` commands in our main `Makefile`?

Reply:
sure thang :)